app/src/main/res/layout/fragment_mentorship_tasks.xml

```diff
@@ -114,7 +114,10 @@
         android:layout_height="wrap_content"
         android:layout_alignParentEnd="true"
         android:layout_alignParentBottom="true"
-        android:layout_margin="10dp"
+        android:layout_marginTop="10dp"
+        android:layout_marginRight="10dp"
+        android:layout_marginLeft="10dp"
+        android:layout_marginBottom="155dp"
```

Comment:
I think hardcoded values should be replaced and can be put into dimnes XML file. Just a suggestion to improve the coding style and increase the ease.